README.md

```diff
@@ -34,9 +34,12 @@ This addon consists of 1 component and 1 service.
 
 The `NavigationGuard` component takes a boolean `@shouldGuard` and an optional string `@message`.
 
-hbs
-```
-<NavigationGuard @shouldGuard={{true}} @message="This component is preventing navigation" />
+```handlebars
+{{!-- app/components/my-component.hbs --}}
```

Comment:
I wasn't too sure where the `<NavigationGuard>` component is supposed to go. I assumed that the location doesn't matter.